openbb_platform/providers/fmp/openbb_fmp/models/form_13f.py

```diff
@@ -0,0 +1,109 @@
+"""Form 13f Model."""
+
+import asyncio
+from datetime import date as dateType
+from typing import Any, Dict, List, Optional
+from warnings import warn
+
+from openbb_core.provider.abstract.fetcher import Fetcher
+from openbb_core.provider.standard_models.form_13FHR import (
+    Form13FHRData,
+    Form13FHRQueryParams,
+)
+from openbb_core.provider.utils.errors import EmptyDataError
+from openbb_core.provider.utils.helpers import amake_request
+from openbb_fmp.utils.helpers import create_url
+from pydantic import Field
+
+
+class FMPForm13FHRQueryParams(Form13FHRQueryParams):
+    """Form 13f Query Parameters.
+
+    Source: https://financialmodelingprep.com/api/v3/form-thirteen/0001388838?date=2021-09-30
+    """
+
+
+class FMPForm13FHRData(Form13FHRData):
+    """Form13 FHR Data Model."""
+
+    __alias_dict__ = {
+        "period_ending": "date",
+        "issuer": "nameOfIssuer",
+        "principal_amount": "shares",
+        "asset_class": "titleOfClass",
+        "filling_date": "fillingDate",
+        "accepted_date": "acceptedDate",
+        "ticker_cusip": "tickercusip",
+        "final_link": "finalLink",
+    }
+    filling_date: dateType = Field(
```

Comment:
Where fields have `default=None`, they need to be annotated as, `Optional[type]`